packages/terra-form-checkbox/src/Checkbox.jsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Checkbox.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+    * The checked property of the Input element. Use this to generate a controlled Checkbox Element.
+    */
+  checked: PropTypes.bool,
```

Comment:
Is checked a first-hand input attribute? If not, I feel like this should be isChecked.

Reply:
[Yeah](https://www.w3schools.com/tags/att_input_checked.asp).